[2025-06-29T12:02:19.830+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-08-31T00:00:00+00:00 [queued]>
[2025-06-29T12:02:19.884+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-08-31T00:00:00+00:00 [queued]>
[2025-06-29T12:02:19.885+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-06-29T12:02:19.920+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-08-31 00:00:00+00:00
[2025-06-29T12:02:19.968+0000] {standard_task_runner.py:57} INFO - Started process 5848 to run task
[2025-06-29T12:02:20.004+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2022-08-31T00:00:00+00:00', '--job-id', '1227', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpm570w1nc']
[2025-06-29T12:02:20.041+0000] {standard_task_runner.py:85} INFO - Job 1227: Subtask insert_into_table
[2025-06-29T12:02:20.354+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-08-31T00:00:00+00:00 [running]> on host f561bfe7dd3a
[2025-06-29T12:02:20.673+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-08-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-08-31T00:00:00+00:00'
[2025-06-29T12:02:20.679+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-08-31', 'dag_with_postgres_operator_v03')
        
[2025-06-29T12:02:20.719+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T12:02:20.749+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T12:02:20.773+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-08-31', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-06-29T12:02:20.777+0000] {sql.py:427} INFO - Rows affected: 1
[2025-06-29T12:02:20.802+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20220831T000000, start_date=20250629T120219, end_date=20250629T120220
[2025-06-29T12:02:20.886+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-29T12:02:21.006+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
